src/BookSettings.ts

```diff
@@ -210,23 +210,23 @@ export default class BookSettings {
         }
     }
 
-    private updateFontSizeButtons(): void {
-        const currentFontSizeIndex = this.fontSizes.indexOf(this.selectedFontSize);
-
-        if (currentFontSizeIndex === 0) {
-            this.fontSizeButtons["decrease"].className = "decrease disabled";
-        } else {
-            this.fontSizeButtons["decrease"].className = "decrease";
-        }
-
-        if (currentFontSizeIndex === this.fontSizes.length - 1) {
-            this.fontSizeButtons["increase"].className = "increase disabled";
-        } else {
-            this.fontSizeButtons["increase"].className = "increase";
-        }
-
-        this.fontSizeLabel.style.fontSize = this.selectedFontSize;
-    }
+    // private updateFontSizeButtons(): void {
+    //     const currentFontSizeIndex = this.fontSizes.indexOf(this.selectedFontSize);
+    //
+    //     if (currentFontSizeIndex === 0) {
+    //         this.fontSizeButtons["decrease"].className = "decrease disabled";
+    //     } else {
+    //         this.fontSizeButtons["decrease"].className = "decrease";
+    //     }
+    //
+    //     if (currentFontSizeIndex === this.fontSizes.length - 1) {
+    //         this.fontSizeButtons["increase"].className = "increase disabled";
+    //     } else {
+    //         this.fontSizeButtons["increase"].className = "increase";
+    //     }
+    //
+    //     this.fontSizeLabel.style.fontSize = this.selectedFontSize;
```

Comment:
you took out a lot more code than you needed to. you could've removed only this last line that sets the font size.